src/Controller/AuthenticationController.php

```diff
@@ -191,17 +201,25 @@ public function authenticationStatusAction()
      *
      * @return bool
      */
-    private function authenticationChallengeIsExpired()
+    private function authenticationChallengeIsExpired(): bool
     {
-        return $this->tiqrService->authenticationUrl() === false;
+        // TODO: This is a hack
```

Comment:
1. Do you suggest we add an additional method on the tiqrService that makes this test more verbose? 
2. I propose you remove the `todo` portion, of the otherwise very informative comment, from the code. Moving it to Pivotal as a story to pick up a later point in time.

Reply:
https://www.pivotaltracker.com/story/show/183080722